agent/setup.py

```diff
@@ -42,20 +42,19 @@
     setupcfg.close()
 
 # Find the correct readme to build into the package based on OS distribution
-release_readme = '/docs/release_readme.rst'
+release_readme = '/release_readme.rst'
 if 'install' in sys.argv: # exact match on arguments
     if any('rpm' in argv for argv in sys.argv):
-        release_readme = '/docs/release_rpm.rst'
+        release_readme = '/release_rpm.rst'
     else:
-        release_readme = '/docs/release_deb.rst'
+        release_readme = '/release_deb.rst'
 
 data_files = [('/usr/bin',
                [project_dir + '/agent/usr/bin/f5-oslbaasv1-agent']),
               ('/etc/neutron',
                [project_dir + '/agent/etc/neutron/f5-oslbaasv1-agent.ini']),
               ('/usr/share/doc/f5-oslbaasv1-agent',
-               [project_dir + '/docs/f5-oslbaasv1-readme.rst',
-                project_dir + release_readme,
+               [project_dir + '/release_readme.rst',
```

Comment:
You need to change this line back to match the old code to grab the OS-specific version.
project_dir + release_readme